lib/controllers/favorites_controller.js

```diff
@@ -56,6 +56,10 @@ async function invalidResponse(response) {
   response.status(400).json({ message: "Invalid request body" })
 }
 
+const noRecordResponse = () => {
+  return { message: "Favorite with that ID does not exist!" }
+}
+
```

Comment:
I like this! Also if you have a different way of phrasing the message let me know.